1020 add the installation page in the quick start guide (#1277)
onflow/fcl-js: docs/reference/installation.mdx

# Installation

> This chapter explains the installation of the FCL library in your system. However, before moving to the installation, let us verify the prerequisite first.

Reviewer: ```suggestion
This chapter explains the installation of the FCL JS library in your system. However, before moving to the installation, let us verify the prerequisite first.
```
```suggestion
This chapter explains the installation of the FCL JS library in your system. However, before moving to the installation, let us verify the prerequisite first.
```

## Prerequisite
- Node.js version v12.0.0 or higher. 

> FCL JS depend on Node.js and must be installed in your system. But if you have installed Node.js on your system. You can check it using the below command −

Reviewer: ```suggestion
FCL JS depends on Node.js version v12.0.0 or higher. You can check your currently installed version using the below command:
```

```javascript
node --version
```

> If Nodejs is not installed on your system, you can download and install it by visiting [Nodejs Download](https://nodejs.org/en/download/). 

Reviewer: ```suggestion
If Node.js is not installed on your system, you can download and install it by visiting [Node.js Download](https://nodejs.org/en/download/). 
```

# Installation

This chapter explains the installation of the FCL library in your system. However, before moving to the installation, let us verify the prerequisite first.

## Prerequisite

> - You need to have Node version v12.0.0 or higher.

Reviewer: ```suggestion
```

This part seems a bit long winded and can be streamlined.

```suggestion
FCL JS depends on Node.js version v12.0.0 or higher. You can check you have the correct version installed using the below command.
```

FCL JS depend on Node.js and must be installed in your system. But if you have installed Node.js on your system. You can check it using the below command −

```javascript
node --version
```

> You will see the Nodejs version you installed, as shown below. The version below is mine.
> 
> ```javascript
> v14.2.0
> ```

Reviewer: ```suggestion
```

If Nodejs is not installed on your system, you can download and install it by visiting [Nodejs Download](https://nodejs.org/en/download/). 

> To use the FCL JS in your application, install using **yarn** or **npm**

Reviewer: ```suggestion
Install FCL JS using **npm** or **yarn**
```
